app/controllers/backgrounds_controller.rb

```diff
@@ -115,6 +115,6 @@ def set_background
 
   # Never trust parameters from the scary internet, only allow the white list through.
   def background_params
-    params.require(:background).permit(:name, pipeline_run_ids: [])
+    params.require(:background).permit(:name, pipeline_run_ids: [], sample_ids: [])
```

Comment:
I fixed the test by fixing another bug, this one in the controller which was dropping the `sample_ids` that was expected by `assign_attributes_and_has_changed?`.